modules/client-configuration/pages/clients-rh.adoc

```diff
@@ -252,6 +260,13 @@ From the command prompt, add [systemitem]``rhel-x86_64-server-6`` and [systemite
 From the {webui}, add [systemitem]``RHEL7 Base x86_64`` and [systemitem]``SUSE Linux Enterprise Client Tools RES7 x86_64``.
 +
 From the command prompt, add [systemitem]``rhel-x86_64-server-7`` and [systemitem]``res7-suse-manager-tools-x86_64``.
++
+* For {es} 8:
++
+From the {webui}, add [systemitem]``RHEL8 Base x86_64`` and [systemitem]``SUSE Linux Enterprise Client Tools RES8 x86_64``.
+// check what the correct name for the AppStream channel is. LKB 2019-12-13
++
+From the command prompt, add [systemitem]``rhel-x86_64-server-8`` and [systemitem]``res8-suse-manager-tools-x86_64``.
```

Comment:
Not sure about this.

When I am right, this chapter is for nativ redhat but the var  {es} is used (Expanded support?)
Please check the displayed titles with the reality.  Afaik the tools product for v8 is named different
Please also check differences betwen products and channels. The UI enables products. For v8 these are multiple channels and automatic enables also recommended products.

Reply:
we note there that `Your {susemgr} subscription entitles you to the tools channels for {sleses} (also known as {redhat} Expanded Support or RES).`